utils/subscription.ts

```diff
@@ -11,3 +15,25 @@ export const serviceOwnerCheckTask = (
     (svcId: NonEmptyString) => svcId === ownerSubscriptionId,
     _ => ResponseErrorForbiddenNotAuthorized
   )(serviceId);
+
+/**
+ * This check verifies that:
+ *  - isVisible === false || (isVisible === true && svcMetadata !== undefined)
```

Comment:
Try to avoid write code into comments. Let's explain the function's purpose using a discursive approach.